weaviate/weaviate_classes.py

```diff
@@ -64,29 +51,16 @@
     "LinkTo",
     "LinkToMultiTarget",
     "MetadataQuery",
-    "Multi2VecBindConfig",
-    "Multi2VecBindConfigWeights",
-    "Multi2VecClipConfig",
-    "Multi2VecClipConfigWeights",
     "ReferenceProperty",
     "ReferencePropertyMultiTarget",
     "Property",
-    "Ref2VecCentroidConfig",
     "Reference",
     "ReferenceTo",
     "ReferenceToMultiTarget",
     "ShardingConfigCreate",
     "StopwordsCreate",
     "StopwordsUpdate",
     "Tenant",
-    "Text2VecAzureOpenAIConfig",
-    "Text2VecCohereConfig",
-    "Text2VecGPT4AllConfig",
-    "Text2VecHuggingFaceConfig",
-    "Text2VecHuggingFaceConfigOptions",
-    "Text2VecOpenAIConfig",
-    "Text2VecPalmConfig",
-    "Text2VecTransformersConfig",
     "Tokenization",
     "Vectorizer",
     "VectorizerConfig",
```

Comment:
factory is missing here and I think we can remove Vectorizer and VectorizerConfig

Reply:
I think `VectorizerConfig` is required for them when using a custom vectorizer, e.g. some standalone Docker container or personal API

Reply:
`VectorizerFactory` is already there in un-diffed code on L15 ;)

Reply:
Ahhh, you're quite right! The type of `VectorizerConfig.vectorizer` should be `Union[Vectorizer, str]` to allow for users to specify their own named vectorizer as a raw string. WDYT?

Reply:
let's remove it for now and leave support for custom modules it for later. We might also need to add a way to set a moduleconfig